catroid/src/main/java/org/catrobat/catroid/content/actions/ChangeSizeByNAction.kt

```diff
@@ -20,36 +20,37 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-package org.catrobat.catroid.content.actions;
-
-import android.util.Log;
-
-import com.badlogic.gdx.scenes.scene2d.actions.TemporalAction;
-
-import org.catrobat.catroid.content.Scope;
-import org.catrobat.catroid.formulaeditor.Formula;
-import org.catrobat.catroid.formulaeditor.InterpretationException;
-
-public class ChangeSizeByNAction extends TemporalAction {
-
-	private Scope scope;
-	private Formula size;
-
-	@Override
-	protected void update(float percent) {
-		try {
-			Float newSize = size == null ? Float.valueOf(0f) : size.interpretFloat(scope);
-			scope.getSprite().look.changeSizeInUserInterfaceDimensionUnit(newSize);
-		} catch (InterpretationException interpretationException) {
-			Log.d(getClass().getSimpleName(), "Formula interpretation for this specific Brick failed.", interpretationException);
-		}
-	}
-
-	public void setScope(Scope scope) {
-		this.scope = scope;
-	}
-
-	public void setSize(Formula size) {
-		this.size = size;
-	}
-}
+package org.catrobat.catroid.content.actions
+
+import android.util.Log
+import org.catrobat.catroid.content.Scope.sprite
+import com.badlogic.gdx.scenes.scene2d.actions.TemporalAction
+import org.catrobat.catroid.content.Scope
+import org.catrobat.catroid.formulaeditor.Formula
+import org.catrobat.catroid.formulaeditor.InterpretationException
+
+class ChangeSizeByNAction : TemporalAction() {
+    private var scope: Scope? = null
```

Comment:
For these simple member variables you can remove the "private" keyword (since they need to be accessible from outside anyway). Then the compiler will also automatically create the getters/setters for you.